app/controllers/dashboard/dashboard-activity-controller.js

```diff
@@ -171,3 +177,35 @@ function getTransactionDateRange (period) {
 
   return { fromDateTime, toDateTime }
 }
+
+async function getStripeAccountDetails (paymentProvider, gatewayAccountId, correlationId) {
+  if (paymentProvider === 'stripe') {
+    try {
+      const stripeResponse = await connector.getStripeAccount(gatewayAccountId, correlationId)
+      const stripeAccoundId = stripeResponse.stripeAccountId
+
+      try {
+        const fullStripeAccountDetails = await retrieveAccountDetails(stripeAccoundId)
+
+        const reducedStripeAccountDetails = {
```

Comment:
`reduced` is a slightly overloaded label here as it usually means something that has been passed through a `<Iterator>.reduce()` call. 

The main thing here is to use something else, maybe `formattedStripeAccount`? 

This could potentially be streamlined for readability with something like: 
```js 
const stripeAccount = await retrieveStripeAccount(stripeAccountId)
const currentDeadline = stripeAccount.requirements.current_deadline
const formattedStripeAccount = { 
  payments_enabled: stripeAccount.payments_enabled,
  current_deadline: currentDeadline && moment.unix(currentDeadline).format('DD/MM/YYYY')
}
```

Reply:
Done